.Documentation/articles/getting-started/evaluator.md

```diff
@@ -59,9 +59,9 @@ All `Advanced` scenes use a custom trainer's overlay and wait for you to click a
 
 If you never used Unity before, you can learn basics [here](https://docs.unity3d.com/Manual/UsingTheEditor.html).
 
-To open a training course for the current scene, select the `[TRAINING_CONFIGURATION]` game object in the [Hierarchy](https://docs.unity3d.com/Manual/Hierarchy.html) view and click `Open course in Workflow Editor`. It will open the Workflow Editor window.
+To open a training course for the current scene, select the `[TRAINING_CONFIGURATION]` game object in the [Hierarchy](https://docs.unity3d.com/Manual/Hierarchy.html) view and click `Open course in Workflow window`. It will open the Workflow window.
```

Comment:
```suggestion
To open a training course for the current scene, select the `[TRAINING_CONFIGURATION]` game object in the [Hierarchy](https://docs.unity3d.com/Manual/Hierarchy.html) view and click `Open course in Workflow Window`. It will open the Workflow window.
```

Reply:
like before, I consider this a sentence on a button.